ios/RNRadar.m

```diff
@@ -730,19 +730,16 @@ - (void)didLogMessage:(NSString *)message {
     }
 
     NSDictionary *nearDict = optionsDict[@"near"];
-    if (nearDict == nil || ![nearDict isKindOfClass:[NSDictionary class]]) {
-        if (reject) {
-            reject([Radar stringForStatus:RadarStatusErrorBadRequest], [Radar stringForStatus:RadarStatusErrorBadRequest], nil);
-        }
-
-        return;
+    CLLocation *near = nil;
+    if (nearDict && [nearDict isKindOfClass:[NSDictionary class]]) {
+        double latitude = [RCTConvert double:nearDict[@"latitude"]];
+        double longitude = [RCTConvert double:nearDict[@"longitude"]];
+        NSDate *timestamp = [NSDate new];
+        near = [[CLLocation alloc] initWithCoordinate:CLLocationCoordinate2DMake(latitude, longitude) altitude:-1 horizontalAccuracy:5 verticalAccuracy:-1 timestamp:timestamp];
```

Comment:
just curious, what is the need for setting altitude, horizontalAccuracy, and verticalAccuracy?

Reply:
It's required for CLLocation: https://developer.apple.com/documentation/corelocation/cllocation/3524339-initwithcoordinate